src/views/admin/plans/components/CustomPlan.js

```diff
@@ -1,11 +1,17 @@
 import React from "react";
 import {Button, Flex, Text} from "@chakra-ui/react";
+import {MdCall} from 'react-icons/md'
+
+const CustomPlan = (props) => {
+    const {
+        setContact,
+    } = props;
 
-const CustomPlan = () => {
     return (
-        <Flex justify="center" mt={"30px"} >
+        <Flex justify="center" mt={"30px"}>
             <Text fontSize='3xl'>Wish to make a custom plan?</Text>
```

Comment:
we should add <form> tag and create function that submit the user contact request and save it into the DB